gluster-utilization.libsonnet

```diff
@@ -0,0 +1,66 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gluster-utilization',
+        rules: [
+          {
+            alert: 'GlusterVolumeUtilization',
+            expr: |||
+                  100 * gluster:volume_capacity_used_bytes_total:sum > 80,
+            |||,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              message: 'Gluster Volume Utilization more than 80%',
+            },
+          },
+          {
+            alert: 'GlusterVolumeUtilization',
+            expr: |||
+                  100 * gluster:volume_capacity_used_bytes_total:sum > 90,
+            |||,
+            'for': '5m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              message: 'Gluster Volume Utilization more than 90%',
+            },
+          },
+          {
+            alert: 'GlusterBrickUtilization',
+            expr: |||
+                  100 * gluster_brick_capacity_used_bytes
```

Comment:
Something similar is required for volume capacity alerts as well (division by total no of bytes)

Reply:
ack.